src/fmu/tools/rms/qcreset.py

```diff
@@ -0,0 +1,383 @@
+    value = config["value"]
+
+    # HORIZON DATA
+    if "horizons" in config.keys():
+        print("Set horizons values to " + str(value) + "...")
+        _set_surfaces_value(project, "horizons", config["horizons"], value)
+
+    # ZONE DATA
+    if "zones" in config.keys():
+        print("Set zones values to " + str(value) + "...")
+        _set_surfaces_value(project, "zones", config["zones"], value)
+
+    # GRID MODEL DATA
+    if "grid_models" in config.keys():
+        print("Set 3D grid properties values to " + str(value) + "...")
+        if isinstance(config["grid_models"], list):
+            # work directly at grid models level
+            for gridname in config["grid_models"]:
+                print(" >> " + gridname)
+                grid = project.grid_models[gridname]
+                for prop in grid.properties:
+                    try:
+                        prop.set_values(prop.get_values() * 0 + value)
+                        print(" >> >> " + prop.name)
+                    except Exception as e:
+                        print(" >> >> " + prop.name + " is already empty")
+                        print(e)
+        elif isinstance(config["grid_models"], dict):
+            # check setup for each grid models (list vs. all)
+            gridnames = config["grid_models"].keys()
+            for gridname in gridnames:
+                print(" >> " + gridname)
+                grid = project.grid_models[gridname]
+                propnames = config["grid_models"][gridname]
+                if isinstance(propnames, str):
+                    if propnames == "all":
+                        for prop in grid.properties:
+                            try:
+                                prop.set_values(prop.get_values() * 0 + value)
+                                print(" >> >> " + prop.name)
+                            except Exception as e:
+                                print(" >> >> " + prop.name + " is already empty")
+                                print(e)
+                    else:
+                        raise Exception(
+                            "keyword " + propnames + "not recognized, 'all' expected!"
+                        )
+                elif isinstance(propnames, list):
+                    for propname in propnames:
+                        try:
+                            prop = grid.properties[propname]
+                            prop.set_values(prop.get_values() * 0 + value)
+                            print(" >> >> " + prop.name)
+                        except Exception as e:
+                            print(" >> >> " + prop.name + " is already empty")
+                            print(e)
+        else:
+            raise TypeError(
+                "Value associated with key 'zones' must be of type list or dict!"
+            )
+
+    print("End of function set_data_constant().")
+
+
+def set_data_empty(config: Dict):
+    """Set data from RMS empty.
+
```

Comment:
Why  use input `config: Dict? ` Should it not only use keywords as given in the docstring?